sympy/sets/sets.py

```diff
@@ -278,10 +278,10 @@ def contains(self, other):
 
         """
         other = sympify(other, strict=True)
-        ret = self._contains(other)
+        ret = sympify(self._contains(other))
```

Comment:
I don't think there would be anything wrong with the ``sympify`` call. It seems that ``print``  is not working correctly with ``S.Complexes`` because it falls back to ``Basic`` printer. It (or possibly ``ComplexRegion``) should probably have its own printer defined.